sql/tables/AccountEarnedCMECredit.sql

```diff
@@ -11,3 +11,9 @@ create index AccountEarnedCMECredit_account_index
 
 create index AccountEarnedCMECredit_credit_index
 	on AccountEarnedCMECredit(credit);
+
+create index AccountEarnedCMECredit_earned_date_index
```

Comment:
I think so. I think we should also create the same type of index for `Subscription.expiry_date` and `Orders.createdate`. It might not have a 5s - 20ms effect, but it's sure to make things more efficient.